pysurveycto/pysurveycto.py

```diff
@@ -724,3 +727,70 @@ def list_forms(self):
             raise e
         
         return response.json()['forms']
+
+    def upload_dataset(self, df, dataset_id, dataset_title=None, append=False, fill=False):
+        """
+        Uploads a pandas df to a dataset
+        :return: dictionary with previous dataset preview and upload response
+
+        :param df: pandas DataFrame to upload
+        :param dataset_id: ID of the dataset on SurveyCTO
+        :param dataset_title: Optional title for the dataset (defaults to dataset_id)
+        :param append: If True, appends data; otherwise replaces the dataset
+        :param fill: If True, allows mismatched columns in append mode
```

Comment:
Looks like this parameter isn't getting used in the function. Can we remove it? 
In case of mismatched columns does the API return an error?